src/main/java/net/dries007/tfc/api/recipes/anvil/AnvilRecipeSplitting.java

```diff
@@ -68,8 +68,7 @@ public boolean matches(ItemStack input)
                     IForgeable cap = dump.getCapability(CapabilityForgeable.FORGEABLE_CAPABILITY, null);
                     if (cap instanceof IForgeableMeasurableMetal)
                     {
-                        cap.setWork(0); //Reset work without resetting temp
```

Comment:
Does this not apply or is it already handled?

Reply:
`cap.reset()` does everything: reset steps, reset recipe and also set work to 0.